link_checker.py

```diff
@@ -460,6 +460,29 @@ def output_summary(all_links, num_errors):
             output_write(url)
 
 
+def output_test_summary(errors_total):
+    """Prints summary of script output in form of junit-xml
+
+    Args:
+        errors_total (int): Total number of broken links
+    """
+    if not os.path.isdir("test-summary"):
+        os.mkdir("test-summary")
+    with open("test-summary/junit-xml-report.xml", "w") as test_summary:
```

Comment:
I recommend using something like https://github.com/kyrus/python-junit-xml instead of manually writing XML as strings to improve readability and maintainability.

Reply:
Looks like `pytest` can also do this natively https://docs.pytest.org/en/latest/usage.html#creating-junitxml-format-files

Reply:
> Looks like `pytest` can also do this natively https://docs.pytest.org/en/latest/usage.html#creating-junitxml-format-files

The unit tests of the scripts are written with `pytest` and they generate a junitxml type summary files, which can be viewed in Circle-CI